### ISSUE TITLE
Add nullable: true to Ajv config

### ISSUE THREAD

**Issue by jwang-navarik** [seriousme/fastify-openapi-glue#60]
Fastify now supports keyword "nullable" from Open API 3 specification and it's set in its baseline ajv configuration: https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md. Can we have something like that in index.js:

```
  const ajv = new Ajv({
    // the fastify defaults
    removeAdditional: true,
    useDefaults: true,
    coerceTypes: true,
    nullable: true, // add this line
  });
```

Thanks in advance.

**Comment by seriousme** [comment]
Hi,

Added and published to NPM as version 1.4.0.
https://www.npmjs.com/package/fastify-openapi-glue

Kind regards,
Hans

**Comment by jwang-navarik** [comment]
Great, many thanks!
    On Tuesday, January 21, 2020, 10:21:55 a.m. PST, Hans Klunder <notifications@github.com> wrote:  
 
 
Hi,

Added and published to NPM as version 1.4.0.
https://www.npmjs.com/package/fastify-openapi-glue

Kind regards,
Hans
